verl/trainer/ppo/ray_trainer.py

```diff
@@ -274,6 +274,9 @@ def __init__(self,
         self.use_rm = Role.RewardModel in role_worker_mapping
         self.ray_worker_group_cls = ray_worker_group_cls
         self.validation_generations_logger = ValidationGenerationsLogger()
```

Comment:
can we pass the dir directly to ValidationGenerationsLogger & RolloutLogger? no need to store the dir in trainer